docs/resource/README.md

```diff
@@ -22,6 +22,11 @@ This document defines standard attributes for resources. These attributes are ty
   - [Semantic Attributes with Dedicated Environment Variable](#semantic-attributes-with-dedicated-environment-variable)
   - [Semantic Attributes with SDK-provided Default Value](#semantic-attributes-with-sdk-provided-default-value)
 - [Service](#service)
+  - [Service in Kubernetes](#service-in-kubernetes)
+    - [How `service.namespace` is calculated](#how-servicenamespace-is-calculated)
```

Comment:
Since this is currently applicable to auto-instrumentation with operator, I think we should clarify that these are the guidelines suggested by conventions and adopted in operator. So maybe change the heading to

```suggestion
    - [How `service.namespace` can be calculated](#how-servicenamespace-can-be-calculated)
```

Reply:
good point

maybe "should" is better?